spec/models/post_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:title) }
+
+    it 'should validate the comments_counter is greater than or equal to 0' do
```

Comment:
- [ ] dear @eudondian, As it have been mentioned by the previous reviewer, and as stated in the project requirement, we need to test all the custom methods in the models, `THIS INCLUDES THE ONE YOU HAVE USED FOR VALIDATION`, Please add a test for `title_not_exceed_250_chars`, `comments_counter_greater_than_zero`, `likes_counter_greater_than_zero` and `increment_posts_counter` methods inside the `Post` model.